sdk/translation/Azure.AI.Translation.Text/api/Azure.AI.Translation.Text.netstandard2.0.cs

```diff
@@ -0,0 +1,262 @@
+        public static bool operator !=(Azure.AI.Translation.Text.ProfanityAction left, Azure.AI.Translation.Text.ProfanityAction right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ProfanityMarker : System.IEquatable<Azure.AI.Translation.Text.ProfanityMarker>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ProfanityMarker(string value) { throw null; }
+        public static Azure.AI.Translation.Text.ProfanityMarker Asterisk { get { throw null; } }
+        public static Azure.AI.Translation.Text.ProfanityMarker Tag { get { throw null; } }
+        public bool Equals(Azure.AI.Translation.Text.ProfanityMarker other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.AI.Translation.Text.ProfanityMarker left, Azure.AI.Translation.Text.ProfanityMarker right) { throw null; }
+        public static implicit operator Azure.AI.Translation.Text.ProfanityMarker (string value) { throw null; }
+        public static bool operator !=(Azure.AI.Translation.Text.ProfanityMarker left, Azure.AI.Translation.Text.ProfanityMarker right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class SentenceLength
+    {
+        internal SentenceLength() { }
+        public System.Collections.Generic.IReadOnlyList<int> SrcSentLen { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<int> TransSentLen { get { throw null; } }
+    }
+    public partial class SourceDictionaryLanguage
+    {
+        internal SourceDictionaryLanguage() { }
+        public string Dir { get { throw null; } }
+        public string Name { get { throw null; } }
+        public string NativeName { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.AI.Translation.Text.TargetDictionaryLanguage> Translations { get { throw null; } }
+    }
+    public partial class SourceText
+    {
+        internal SourceText() { }
+        public string Text { get { throw null; } }
+    }
+    public partial class TargetDictionaryLanguage
+    {
+        internal TargetDictionaryLanguage() { }
+        public string Code { get { throw null; } }
+        public string Dir { get { throw null; } }
+        public string Name { get { throw null; } }
+        public string NativeName { get { throw null; } }
+    }
+    public partial class TextTranslationClient
```

Comment:
I see that most (if not all) of these methods have a really large number of parameters. These methods must adopt the "options" pattern. You can find the guidelines for service method parameters [here](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-parameters).

Reply:
It was discussed earlier and will be done for GA.